index.php

```diff
@@ -6,28 +6,33 @@
  * @license https://www.humhub.com/licences
  */
 
-// comment out the following two lines when deployed to production
```

Comment:
This needs to be updated in the documentation here: https://docs.humhub.org/docs/admin/security/#enable-production-mode

Reply:
@dreua Thank you, we will adapt the documentation accordingly.
In future, debug mode will be automatically deactivated after installation.